### DISCUSSION TITLE
504 Gateway Timeout Error on Vercel

### DISCUSSION THREAD

**Discussion by piotrkulpinski** [nextauthjs/next-auth#8944] [Help]
### Summary

Hi, 

I'm trying to deploy the app to Vercel but the authentication made using NextAuth is not working. 

Whenever I click on the social login button, it's keep loading until the 15sec limit is reached and then it crushes with 504 - Gateway Timeout error. 

Any idea what could be the reason? 

I have the env variables set up.

### Additional information

```js
NextJS version: 13.5.5
NextAuth version: 0.0.0-manual.b53ca00b
```

**Comment by brunomarks7** [comment]
Im getting the same error right now in Next 13 with NextAuth.

**Comment by mununki** [comment]
I'm getting the same issue in Next 13 with NextAuth 4.
Anybody resolves this issue?

**Comment by RobertConde** [comment]
Me too 😢

**Comment by takavor** [comment]
Same issue
